cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -64,10 +65,11 @@ func GetResourceTypeDisplayName(resourceType AzureResourceType) string {
 		return "Azure API Management"
 	case AzureResourceTypeCacheForRedis:
 		return "Cache for Redis"
-	case AzureResourceTypeSqlDatabase:
-		return "Azure SQL DB"
+	case AzureResourceTypeSqlServer:
+		return "Azure SQL Server"
+	case AzureResourceTypePostgreSqlServer:
+		return "Azure PostgreSQL Server"
```

Comment:
There's also a "Standard server" so I could have specified "Flexible server" here, but from what I understand, standard servers are on the way out so I don't know that azd needs to support them. (i.e. flexible server new hotness)

Reply:
Don't really have a strong opinion -- can go either way, but "Azure Database for PostgreSQL flexible server" is the translation provided by Portal currently.

Reply:
Which is a real mouthful but that's what the service owners decided on..